src/FFMpeg/FFProbe/OptionsTester.php

```diff
@@ -42,7 +42,7 @@ public function has($name)
 
         $output = $this->retrieveHelpOutput();
 
-        $ret = (Boolean) preg_match('/^'.$name.'/m', $output);
+        $ret = (bool) preg_match('/^'.$name.'/m', $output);
```

Comment:
just noticed I replaced that one here as well and learned, that Boolean is a valid Type in PHP. But as it is quite uncommon and looks like a class, and Psalm also considers it a class, I still suggest to merge these changes.

Reply:
@digilist According to [this page](https://www.php.net/manual/en/language.types.boolean.php), this looks like a legit way to force conversion to boolean type.